### PR TITLE
e2e: Enterprise Contract

### PR DESCRIPTION
**Pull request by subhashkhileri**
## Fixes 

https://issues.redhat.com/browse/HAC-4046

## Description

**Create A new Integration Test using the image bundle**

Make a commit and validate the EC on pipeline run details.

Validate **Logs** Tab.
Validate **Security** Tab : check Result summary, Rules are displayed and check `Blocking CVE check` is failed and verify its Message is displayed.

Add above Test case in **Advance flow**.

## Type of change

- [ ] Feature
- [ ] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Browser conformance: 

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Comment by subhashkhileri** [comment]
[test]

**Comment by subhashkhileri** [comment]
[test]

**Comment by subhashkhileri** [comment]
[test]

**Comment by subhashkhileri** [comment]
[test]

**Comment by subhashkhileri** [comment]
[test]

**Comment by subhashkhileri** [comment]
[test]

File: integration-tests/support/pages/tabs/PipelinerunsTabPage.ts [hunk 1]
```diff
@@ -19,6 +26,39 @@ export class PipelinerunsTabPage {
...
+          headers: {
+            authorization: `Bearer ${token}`,
+            accept: 'application/json',
+          },
+        };
+        cy.request(request).its('body.items[0].metadata.name');
+      });
+  }
+
+  static verifyECSecurityRuelsResultSummary(summary: RegExp) {
```
Comment (jrichter1) [inline integration-tests/support/pages/tabs/PipelinerunsTabPage.ts:51]: typo in the name

Comment (subhashkhileri) [reply integration-tests/support/pages/tabs/PipelinerunsTabPage.ts]: fixed.

**Pull request merged**